src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/RetrieveDataServiceTest.java

```diff
@@ -53,8 +53,8 @@ class RetrieveDataServiceTest {
     private final ParseRequestService parseRequestService = Mockito.mock(ParseRequestService.class);
 
 
-    RetrieveDataService sutJrdV1 = new RetrieveDataService(parseRequestService, crdService, jrdService, false);
-    RetrieveDataService sutJrdV2 = new RetrieveDataService(parseRequestService, crdService, jrdService, true);
+    RetrieveDataService sutJrdV1 = new RetrieveDataService(parseRequestService, crdService, jrdService, false, false);
```

Comment:
do we need additional sutJrd object and tests to cover 
RetrieveDataService(parseRequestService, crdService, jrdService, true, false);
?

Reply:
Additional tests added in e1a94200377fa298af4b5fba4e595f174c05c216.

Reply:
LGTM